packages/upload-media/src/image-file.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * ImageFile class.
+ *
+ * Small wrapper around the `File` class to hold
+ * information about current dimensions and original
+ * dimensions, in case the image was resized.
+ */
+export class ImageFile extends File {
```

Comment:
I think this is the first class that I see in Gutenberg really. 🤔

Reply:
Can this be just a regular object?

Reply:
It must extend `File` as it is being passed to places expecting a `File`

Reply:
Why is a class bad? RichTextData is also a class that sort of extends String.